packages/models-library/src/models_library/user_preferences.py

```diff
@@ -74,9 +74,10 @@ def get_preference_name(cls) -> PreferenceName:
 
     @classmethod
     def get_default_value(cls) -> Any:
+        # ANE: please double check this concept
         return (
-            cls.model_fields["value"].default_factory()
-            if cls.model_fields["value"].default_factory
+            cls.model_fields["value"].default_factory()  # type: ignore[call-arg]
+            if cls.model_fields["value"].default_factory is not None
```

Comment:
Q: why is this checked against None?

Reply:
Because `default_factory: Callable[[], Any] | Callable[[dict[str, Any]], Any] | None` and you want to check whether a callback is in place. I changed to `callable`